woocommerce/woocommerce
# Disable zoom on touch devices

### All Submissions:






### Changes proposed in this Pull Request:

The zoom function currently works on 'click' but this is inferior to pinch to zoom.

To use pinch to zoom one must open the photoswipe lightbox, however, this has a small click target due to zoom.

This PR thus disables image zoom when using a touch device.

Closes #19688

FWIW in the past we used the zoom script in touch mode so moving your finger would zoom around the image, however, this prevented the user scrolling the page and was disabled.

### How to test the changes in this Pull Request:

1. Open up a product page on mobile.
2. Touching the gallery image should open the lightbox.
3. No magnifier icon should be visible on the main image.

### Changelog entry

> Disable image gallery zoom on touch devices to make photoswipe more easily accessible.
